src/test/java/uk/gov/hmcts/reform/orgrolemapping/domain/service/DroolCivilHearingJudicialRoleMappingTest.java

```diff
@@ -28,16 +28,18 @@
 class DroolCivilHearingJudicialRoleMappingTest extends DroolBase {
 
     String userId = "3168da13-00b3-41e3-81fa-cbc71ac28a69";
-    List<String> judgeRoleNamesWithWorkTypes = List.of("judge", "task-supervisor", "circuit-judge",
-            "specific-access-approver-judiciary", "fee-paid-judge");
+    List<String> judgeRoleNamesWithWorkTypes = List.of("judge", "specific-access-approver-judiciary", "fee-paid-judge");
+
+    List<String> judgeRoleNamesWithExtendedWorkTypes = List.of("circuit-judge", "district-judge", "district-judge",
```

Comment:
"district-judge" is in here twice

Reply:
removed now, apologies, I missed this.

Reply:
Resolved in e88c8c124ed76e7acca7f917670e2097b70ecabe.